pkg/i18n/english.go

```diff
@@ -1122,6 +1125,7 @@ func EnglishTranslationSet() TranslationSet {
 		SetAsUpstream:                       "Set as upstream of checked-out branch",
 		SetUpstream:                         "Set upstream of selected branch",
 		UnsetUpstream:                       "Unset upstream of selected branch",
+		ViewUpstreamResetOptions:            "View reset options targetting selected branch upstream",
```

Comment:
```suggestion
		ViewUpstreamResetOptions:            "Reset checked-out branch onto {{.upstream}}",
```
I think it's okay to drop the 'view options' part so long as it's styled with the ellipsis